src/build.gradle

```diff
@@ -0,0 +1,4 @@
+buildscript {
+    ext.kotlin_version = '1.4.32'
```

Comment:
Not at all suggesting an issue here... but what does this do? It's duplicating something we have in `src/app/build.gradle.kts`, so if this file is useful for IDEA or something then let's try to remove from the other file.

If this file does need to be here then please convert to Kotlin rather than Groovy syntax (incl. adding `.kts` suffix) - I don't really want to re-introduce Groovy after converting the other script!

Reply:
I'm really not sure why, but I don't seem to be able to get IntelliJ to place nicely (both intellisense and gradle integration) without a file at this level. Will change to the kotlin syntax and see what effect that has.

Reply:
Actually, scratch that, it turns out it does work without that file, I think I just hadn't given intellisense time to sort itself out! :facepalm: